src/lib/uri/uri.c

```diff
@@ -9,9 +9,115 @@
 
 #define XSTRNDUP(s, n)  (s != NULL ? xstrndup(s, n) : NULL)
 
+/**
+ * Find URI query param by its @a name in @a uri structure.
```

Comment:
Nit: I wouldn't use abbreviations in comments: `query param` -> `query parameter`.

Reply:
fixed, except places where it is used for doxygen syntax.